spec/requests/api/v1/targets/destroy_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+describe 'Destroy Target', type: :request do
+  subject { delete api_v1_target_path(target), headers:, as: :json }
+
+  let(:user) { create(:user) }
+  let(:target) { create(:target, user:) }
+  let(:headers) { user.create_new_auth_token }
+
+  context 'when success' do
+    it 'should return a sucessfull response' do
+      subject
+      expect(response).to have_http_status(:no_content)
+    end
+  end
```

Comment:
Could you add a test to check that it actually deletes the record?

Reply:
You can check that the Target.amount decrement by 1 or something like that